modular_tannhauser/modules/CitOwOChems/code/chemistry/reagents/furrrrrr.dm

```diff
@@ -23,8 +23,7 @@
 	if(method == INJECT)
 		M.adjustOxyLoss(reac_volume)
 		M.emote("cough")
-		//M.DefaultCombatKnockdown(50)
-		if(reac_volume >= 15)
+		if(reac_volume >= 15) // one normal needle's worth
```

Comment:
is there a way to reference the capacity of a standard syringe? that way if the capacity of the syringe changes the requirement here stays pegged?

Reply:
I'm not sure there's a way without making the syringe's volume a global var, or spawning in a reference syringe every time it gets injected